src/bentoml/_internal/bento/docker/templates/base_alpine.j2

```diff
@@ -5,7 +5,7 @@
 # Install helpers
 RUN {{ common.mount_cache("/var/cache/apk") }} \
     apk add --update bash gcc libc-dev shadow musl-dev build-base \
-		linux-headers g++
+		linux-headers g++ git
```

Comment:
Why do we have to install git into here? can't user just specify it under `system_packages`?

Reply:
after discussion we decide to not include git but make documentation clearer here